src/diy/smol-slime.md

```diff
@@ -73,55 +73,60 @@ Buttons and slide switches are recommended but not required. Buttons can be adde
 
 | IMU + Magnetometer                                 | Product Page                                                                 |
 | -------------------------------------------------- | ---------------------------------------------------------------------------- |
 | [ICM-45686](imu-comparison.md#ICM-45686) + IST8306 | [store.kouno.xyz](https://store.kouno.xyz/products/icm-45686-ist8306-module) |
 | LSM6DSR + IST8306                                  | [store.kouno.xyz](https://store.kouno.xyz/products/lsm6dsr-ist8306-module)   |
 
 ### Buttons
-Push buttons/momentary switches are used to control the tracker. Multiple button configurations are supported. A tracker can have either a reset button, a user button, or both.
+Push buttons and momentary switches are utilized to control the tracker. Multiple button configurations are supported. A tracker can be equipped with either a reset button, a user button, or both.
 
-The reset button is suitable for all functionality. If a user button is available, it will be used instead.
+The reset button is designed to support all functionalities. If an user-specific button is available, it will be utilized instead.
 
-If a button is not available, tweezers can be used to short the pins for initial tracker setup.
+If a button is unavailable, tweezers can be used to short the pins for the initial tracker setup.
 
 ### Switches
-A slide switch can be used to physically disconnect a battery. Some boards have a high standby power draw and will require a switch.
+A slide switch can be used to physically disconnect a battery. Some circuit boards have a high standby power draw and will require a switch.
 
-If a switch is not used, a tracker can enter deep sleep by holding down the user button.
+If a switch is not utilized, a tracker can enter deep sleep mode by pressing and holding down the user-specific button.
 
 ### Batteries
-Most boards will support a 3.7V Li-ion/LiPo battery. Usually, batteries have a maximum charge rate of 1C, or a 1 hour charge rate. Do not use a battery if the charge rating will be exceeded.
-
-To extend the lifespan of the battery, a much lower charge rate close to 0.5C is recommended.
-
-| Board                      | Default charge rate | Minimum battery capacity | Recommended battery capacity |
-| -------------------------- | ------------------- | ------------------------ | ---------------------------- |
-| SuperMini nRF52840         | 100mA               | 100mAh                   | 180-300mAh                   |
-| Seeed Studio XIAO nRF52840 | 50mA                | 50mAh                    | 80-300mAh                    |
+Most boards support a 3.7V Li-ion/LiPo battery. Typically, these batteries have a maximum charge rate of 1C, which corresponds to a one-hour charge time. Do not use a battery if its charge rating will be exceeded.
```

Comment:
is the removed information available somewhere else, or just not needed anymore?